### PR TITLE
Align tile widths

### PR DESCRIPTION
**Pull request by DennisTsiang**
Aligned tile widths so that they are all consistent on the site, and give them a padding on either side.

### CODE REVIEW

File: source/people.erb [hunk 1]
```diff
@@ -14,41 +14,41 @@ email_domain: "unboxed.co"
...
-  <% data.people.each do |person| %>
-    <div class="person-tile">
-      <div class="person-tile__content">
-        <div class="anchor">
-          <span id="<%= person.short_name %>"></span>
-        </div>
-        <div class="person-tile__image-container">
-          <img class="person-tile__image" src="/assets/images/people/<%= person.short_name %>.png" alt="<%= person.name %>" />
-        </div>
+    <% data.people.each do |person| %>
```
Comment (neilvanbeinum) [inline source/people.erb]: I don't think this needs indenting

File: source/assets/stylesheets/styles/_homepage_tile.scss [hunk 2]
```diff
@@ -2,12 +2,15 @@
...
-  padding-right: 6px;
+  padding-right: 0px;
   text-decoration: none;
 
   @include breakpoint($tablet-breakpoint) {
     float: left;
     width: 33.33%;
+    padding-left: 3px;
+    padding-right: 3px;
+
```
Comment (neilvanbeinum) [inline source/assets/stylesheets/styles/_homepage_tile.scss]: Extra line ✂️  :)

**Review by neilvanbeinum — CHANGES_REQUESTED** [review]
Left a couple of very minor comments. Overall I think it looks good. As we discussed in person, I think the product story tiles at tablet width might need to kept more consistent with the existing design.

File: source/assets/stylesheets/styles/_homepage_tile.scss (hunk 2 above)
Comment (DennisTsiang) [reply source/assets/stylesheets/styles/_homepage_tile.scss]: That file gets changed in a later commit (3c49b35)

File: source/assets/stylesheets/styles/_small_tile.scss [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    width: 33%;
+  }
+}
+
+.small-tile__image {
+  background-color: $white;
+  border: 1px solid darken($light-gray, 5%);
+  box-sizing: border-box;
+  display: block;
+  width: 100%;ht: 190px;
```
Comment (neilvanbeinum) [inline source/assets/stylesheets/styles/_small_tile.scss]: The text after the first `;` looks like a typo that can be deleted

File: source/product-stories.erb [hunk 4]
```diff
@@ -89,7 +89,7 @@ product_stories:
         <%= image_tag product_story.image, alt: product_story.client, class: "#{class_name}__image", srcset: retina_srcset(product_story.image) %>
         <div class="<%= class_name %>__content">
           <div class="<%= class_name %>__client"><%= product_story.client %></div>
-          <div class="<%= class_name %>__title"><%= product_story.title %></div>
+          <div class="<%= class_name %>__paragraph"><%= product_story.title %></div>
```
Comment (neilvanbeinum) [inline source/product-stories.erb]: Why did we rename this to `paragraph`? I agree that calling the content `title` isn't accurate, but paragraph is a bit generic to my mind. How about `description`? Whichever name we use I suggest that the class name and the key match in the same way as we've done for `client`.

File: source/assets/stylesheets/styles/_roles_tile.scss [hunk 5]
```diff
@@ -86,6 +86,7 @@
   width: 100%;
   margin: auto;
   object-fit: cover;
```
Comment (neilvanbeinum) [inline source/assets/stylesheets/styles/_roles_tile.scss]: It would be good to add a comment here saying why we've got this unusual rule

**Review by neilvanbeinum — CHANGES_REQUESTED** [review]

File: source/assets/stylesheets/styles/_roles_tile.scss (hunk 5 above)
Comment (DennisTsiang) [reply source/assets/stylesheets/styles/_roles_tile.scss]: Good point, I'll add a comment explaining this in the commit

Comment (neilvanbeinum) [reply source/assets/stylesheets/styles/_roles_tile.scss]: It's good we've got the description in the commit message but I also think a comment in the CSS file itself would be useful. A short one-liner should do it just explaining that the object fit JS library requires this rule to work.

**Review by neilvanbeinum — APPROVED** [review]

File: source/assets/stylesheets/styles/_roles_tile.scss (hunk 5 above)
Comment (DennisTsiang) [reply source/assets/stylesheets/styles/_roles_tile.scss]: Added short comments to the object-fit property now in the last commit

**Pull request merged**